### PR TITLE
local read support inner 1752

### PR DESCRIPTION
**Pull request by ylinzhu**
Reason:  
   Improve 
Type:  
  Improve  
Influences：  
   fix xx

### CODE REVIEW

File: src/main/java/com/actiontech/dble/config/model/SystemConfig.java [hunk 1]
```diff
@@ -1628,4 +1658,23 @@ public boolean isCloseHeartBeatRecord() {
...
+    private void checkChineseProperty(String val, String name) {
+        if (Objects.nonNull(val)) {
+            int length = 11;
+            if (val.length() > length) {
+                problemReporter.warn("Property [ " + name + " ] " + val + " in bootstrap.cnf is illegal，the value contains a maximum of " + length + " characters");
+            }
+            if (StringUtil.isMessyCode(val)) {
+                problemReporter.warn("Property [ " + name + " ] " + val + " in bootstrap.cnf is illegal，the " + Charset.defaultCharset().name() + " encoding is recommended");
+            }
+            String chinese = val.replaceAll(DBConverter.PATTERN_DB.toString(), "");
```
Comment (dcy10000) [inline src/main/java/com/actiontech/dble/config/model/SystemConfig.java:1671]: why replace it?

File: src/main/java/com/actiontech/dble/util/StringUtil.java [hunk 2]
```diff
@@ -609,4 +614,45 @@ public static String trim(String orgStr, char c) {
         }
         return orgStr;
     }
+
+
+    public static boolean isChinese(char c) {
```
Comment (dcy10000) [inline src/main/java/com/actiontech/dble/util/StringUtil.java:619]: add source of code  will be better

File: src/main/java/com/actiontech/dble/util/StringUtil.java [hunk 3]
```diff
@@ -609,4 +614,45 @@ public static String trim(String orgStr, char c) {
...
+            return true;
+        }
+        return false;
+    }
+
+    public static boolean isChinese(String val) {
+        return val.codePoints().allMatch(codepoint -> Character.UnicodeScript.of(codepoint) == Character.UnicodeScript.HAN);
+    }
+
+    public static boolean isMessyCode(String strName) {
```
Comment (dcy10000) [inline src/main/java/com/actiontech/dble/util/StringUtil.java:636]: add source of code will be better

**Review by dcy10000 — CHANGES_REQUESTED** [review]

File: src/main/java/com/actiontech/dble/util/StringUtil.java (hunk 2 above)
Comment (ylinzhu) [reply src/main/java/com/actiontech/dble/util/StringUtil.java:619]: Naming rules are required

File: src/main/java/com/actiontech/dble/config/model/SystemConfig.java (hunk 1 above)
Comment (ylinzhu) [reply src/main/java/com/actiontech/dble/config/model/SystemConfig.java:1671]: Naming rules are required

File: src/main/java/com/actiontech/dble/util/StringUtil.java (hunk 3 above)
Comment (ylinzhu) [reply src/main/java/com/actiontech/dble/util/StringUtil.java:636]: ok

**Review by dcy10000 — APPROVED** [review]
lgtm

**Pull request merged**